lib/models/blueprint.js

```diff
@@ -1017,7 +1017,7 @@ let Blueprint = CoreObject.extend({
     return task.run({
       verbose: true,
       packages: packageNamesAndVersions,
-      installOptions,
+      installOptions: installOptions || { save: true },
```

Comment:
This should be `merge`d. (I'm aware that the original didn't merge and that this is simply moved.)